main_test.go

```diff
@@ -920,6 +920,46 @@ func TestTokenServices(t *testing.T) {
 			t.Fatal("expected an error")
 		}
 	})
+
+	t.Run("init fungible token vaults on account creation", func(t *testing.T) {
+		cfg := test.LoadConfig(t)
+		cfg.InitFungibleVaultsOnAccountCreation = true
+		app := test.GetServices(t, cfg)
+
+		svc := app.GetTokens()
+		accountSvc := app.GetAccounts()
+
+		ctx := context.Background()
+
+		// Make sure FUSD is deployed
+		err := svc.DeployTokenContractForAccount(ctx, true, "FUSD", cfg.AdminAddress)
+		if err != nil {
+			if !strings.Contains(err.Error(), "cannot overwrite existing contract") {
+				t.Fatal(err)
+			}
```

Comment:
Is an error ever expected here? Why not call `t.Fatal(err)` on any error that's returned? Right now it will ignore errors that do not match the above string.

Reply:
This is for test concurrency as FUSD contract will be deployed multiple times. I moved the check into the test helper function: https://github.com/flow-hydraulics/flow-wallet-api/pull/314/commits/d2ab6d86a86f1b8d87aef978b46008bedc4704fe